### PR TITLE
feature(function):Added sample function for softDelete

### PR DESCRIPTION
**Pull request by fivetran-sinchanahegde**
closes https://fivetran.atlassian.net/browse/RD-462313
Added sample function for softDelete support.

### CODE REVIEW

**Review by fivetran-shreyasnemani — APPROVED** [review]
Looks Good. Please test this code on our test function connector before merging.

File: softDelete/aws-lambda.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    response['delete'] = delete
+    #Add all the tables to be soft Deleted in softDelete
+    response['softDelete']=softDelete
+
+    return response
+
+def api_response(state, secrets):
+    # your api call goes here
+    insertTransactions = [
+            {"date":'2017-12-31T06:12:04Z', "order_id":12, "amount":'$1200', "discount":'$13'},
```
Comment (ParagPatil96) [inline softDelete/aws-lambda.py:37]: Kinldy add a random order_id here , as everytime we will get same records and that will create confusion for customer
Also add a comment explaining this

**Comment by fivetran-sinchanahegde** [comment]
Screenshot 2023-05-17 at 3 14 18 PM

**Review by ParagPatil96 — APPROVED** [review]
LGTM

**Pull request merged**